boards/arm/stm32f3_disco/revision.cmake

```diff
@@ -0,0 +1,4 @@
+board_check_revision(
+        FORMAT LETTER
+        DEFAULT_REVISION E
```

Comment:
Please don't change default revision.
Till now people used older revision and may have scripts using it.
Setting the new revision as default is not backward compatible.